apps/poker-roll/src/app/homePage.jsx

```diff
@@ -1,12 +1,40 @@
 import React from "react";
-
+import Outlet from "react-router-dom";
```

Comment:
Double check the docs on how to import components from `react-router-dom`

Reply:
{ Outlet } needs to be destructred to use its methods. is that what you meant for me to fix?